app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,23 @@
+class AddressesController < ApplicationController
+  def create
+    @user = Current.user
+    @address = @user.addresses.create(address_params)
+    redirect_to user_path(@user)
+    flash[:notice] = "Address added Succesfully."
+  end
+
+  def destroy
+    @user = User.find(params[:user_id])
+    @address = @user.addresses.find(params[:id])
```

Comment:
What's the difference between `find` and `find_by`?

Reply:
And what if the record is not found?

Reply:
If record not found exception handled